store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/AddGraph.java

```diff
@@ -178,7 +179,7 @@ public Builder parentSchemaIds(final List<String> parentSchemaIds) {
             return _self();
         }
 
-        public Builder graphAuths(final String... graphAuths) {
+        public Builder setGraphAuths(final String... graphAuths) {
```

Comment:
This is now inconsistent with FederatedAccessHook.Builder.graphAuths. Perhaps just rename it back to graphAuths.

Reply:
My reasoning was
```
.graphAuths("a").graphAuths("b").graphAuths("c").graphAuths("d")
```
Doesn't append, it overwrites. maybe I should try and change how the method is implemented?

Reply:
Undoing after our discussion.